connectors/google/google-sheets/src/main/java/io/camunda/connector/gsheets/model/request/impl/AddValues.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. Licensed under a proprietary license.
+ * See the License.txt file for more information. You may not use this file
+ * except in compliance with the proprietary license.
+ */
+package io.camunda.connector.gsheets.model.request.impl;
+
+import io.camunda.connector.gsheets.model.request.Input;
+import java.util.Objects;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+public class AddValues implements Input {
+
+  @NotBlank private String spreadsheetId;
+  private String worksheetName;
```

Comment:
IMO, both these could be `@Secret`. WDYT?

Reply:
pushed with new commits